plugins/docker-machine/docker-machine.plugin.zsh

```diff
@@ -0,0 +1,20 @@
+# Authors:
+# https://github.com/HaraldNordgren
+#
+# Docker-machine related zsh aliases
+
+alias dm='docker-machine'
+
+alias dmc='docker-machine create'
+alias dmls='docker-machine ls'
+alias dmrestart='docker-machine restart'
+alias dmrm='docker-machine rm'
+alias dmstart='docker-machine start'
+alias dmstop='docker-machine stop'
+alias dmst='docker-machine status'
+alias dmnfs='docker-machine-nfs'
```

Comment:
I like most aliases, I used very similar ones myself, however, `docker-machine-nfs` is not official docker-machine command, I don't think this one should be included.